catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlQueryResponse.java

```diff
@@ -124,14 +125,18 @@ public CqlQueryResponse(
             .getResults()
             .stream()
             .map(
-                result ->
-                    new CqlResult(
-                        result,
-                        searchTerms,
-                        queryResponse.getRequest(),
-                        normalize,
-                        filterAdapter,
-                        actionRegistry))
+                result -> {
```

Comment:
✏️ You could curry the the `CqlResult` construction so that it collapses the map a little:

`Function<Whatever-result-is, CqlResult> cqlResultFunction = result -> new CqlResult(result, searchTerms, queryResponse.getRequest(), normalize, filterAdapter, actionRegisty);`

and then update the map:

`.map(cqlResultFunction)`

Reply:
I partially did this by splitting it up into two different map calls instead